debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/xstream/XStreamStreamingChangeEventSourceMetrics.java

```diff
@@ -18,7 +18,8 @@
 public class XStreamStreamingChangeEventSourceMetrics extends AbstractOracleStreamingChangeEventSourceMetrics {
     public XStreamStreamingChangeEventSourceMetrics(CdcSourceTaskContext taskContext,
                                                     ChangeEventQueueMetrics changeEventQueueMetrics,
-                                                    EventMetadataProvider metadataProvider) {
-        super(taskContext, changeEventQueueMetrics, metadataProvider);
+                                                    EventMetadataProvider metadataProvider,
+                                                    java.util.function.Supplier<java.util.Collection<? extends io.debezium.spi.schema.DataCollectionId>> dataCollectionsSupplier) {
```

Comment:
In some metrics classes we imported `Supplier`, `Collection`, and `DataCollectionId`, but in others we did not. Could we be consistent? I didn't look at all the usages across the metrics classes but I suspect there are likely not any reason we can't import the classes for readability?